### PR TITLE
EasyBlock for Amber14

### PR DESCRIPTION
**Pull request by thiell**
Modified from existing Amber12 EB
Originally installed with EasyBuild 2.3

### CODE REVIEW

**Comment by boegel** [comment]
@thiell is this based on or related to #491?

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by thiell** [comment]
@boegel: Yes indeed, it's based on Amber12 EB found in #491. Maybe not the latest version though. It helped me for installing Amber 14 with EasyBuild. Even if not perfect yet, this may be useful to others and we wanted to share this.

**Comment by boegel** [comment]
@thiell I'll pick up on this, since we just got an install request for Amber+AmberTools too...

I'll take it where you left off, check if there's anything missing from #491, and get it merge-ready.

Thanks a lot for sharing your work!

cc @valtandor

File: easybuild/easyblocks/a/amber.py [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+        return ConfigureMake.extra_options(extra_vars)
+
+    def __init__(self, *args, **kwargs):
+        super(EB_Amber, self).__init__(*args, **kwargs)
+        self.already_extracted = False
+        self.build_in_installdir = True
+
+    def extract_step(self):
+        """Only extract from the tarball if this has not already been done."""
+        if not self.already_extracted:
```
Comment (boegel) [inline easybuild/easyblocks/a/amber.py]: @thiell, @valtandor: any idea why this is here at all?

Comment (thiell) [inline easybuild/easyblocks/a/amber.py]: @boegel you mean the already_extracted? I'm not sure at all sorry :) that doesn't too good indeed.

Comment (boegel) [inline easybuild/easyblocks/a/amber.py]: Yes, I was referring to the `already_extracted` conditional... I guess it's a leftover of earlier attempts by @valtandor.

I'm cleaning up this easyblock a bit, so I'll chuck this and see what happens.

**Comment by boegel** [comment]
merged via #958, thanks @thiell!

**Pull request merged**